docs/docs/05-reusable-components.md

```diff
@@ -72,7 +72,16 @@ React.createClass({
       if (!/matchme/.test(props[propName])) {
         return new Error('Validation failed!');
       }
-    }
+    },
+
+    // Further, you can also supply a custom type checker to `arrayOf`. The
+    // function will be called on each item in the array, The first two
```

Comment:
Nit: Can we make a linebreak before the first `The` so it starts on the next line?
Another nit: `on each time` => `for each item`.
Yet another nit: please change `, The` to `.` and start `The` on the next line it doesn’t “hang” here either.

Thank you!